libs/remix-ui/static-analyser/src/lib/remix-ui-static-analyser.tsx

```diff
@@ -65,7 +65,7 @@ export const RemixUiStaticAnalyser = (props: RemixUiStaticAnalyserProps) => {
 
   useEffect(() => {
     compilation(props.analysisModule, dispatch)
-  }, [])
+  }, [props])
```

Comment:
I think it is better to listen on props.analysisModule instead.

Reply:
I tried, this `useEffect` was not triggered with that too